src/demo-app/data-table/data-table-demo.ts

```diff
@@ -22,4 +22,13 @@ export class DataTableDemo {
     let distanceFromMiddle = Math.abs(50 - progress);
     return distanceFromMiddle / 50 + .3;
   }
+
+  toggleColorColumn() {
+    let colorColumnIndex = this.propertiesToDisplay.findIndex((col: string) => col === 'color');
+    if (colorColumnIndex == -1) {
+      this.propertiesToDisplay.push('color');
```

Comment:
Possibly later, but I'm always cautious about passing strings around because I'm great at mistyping them. Perhaps a best practice would be to define a columns type for the demo:

type columns = 'color'|'progress'; // etc.

And then we'll have a little bit of type safety with the propertiesToDisplay. It would also make us less likely to hit the no column definition error.